### PR TITLE
Remove some unused variables

### PR DESCRIPTION
**Pull request by Geod24**

### CODE REVIEW

**Comment by Geod24** [comment]
Hum, I forgot this is actually far from the full set of unused variables. The dscanner check matches tons of unused variables and parameter, so removed it, but for the moment this is already an improvement that can be reviewed.

**Review by thewilsonator — APPROVED** [review]

**Review by PetarKirov — APPROVED** [review]

**Comment by ZombineDev** [comment]
testpdb fails both on the auto tester and Azure Pipelines, though I don't see why this should be caused by any of the changes in this PR.

File: src/dmd/tocvdebug.d [hunk 1]
```diff
@@ -422,7 +422,6 @@ void toDebug(StructDeclaration sd)
     const len1 = numidx + cv4_numericbytes(cast(uint)size);
     debtyp_t *d = debtyp_alloc(len1 + cv_stringbytes(id));
     cv4_storenumeric(d.data.ptr + numidx, cast(uint)size);
-    const uint len = len1 + cv_namestring(d.data.ptr + len1,id);
```
Comment (rainers) [inline src/dmd/tocvdebug.d]: The call to cv_namestring must not be removed, it is copying the name.

File: src/dmd/tocvdebug.d [hunk 2]
```diff
@@ -557,7 +556,6 @@ void toDebug(ClassDeclaration cd)
     const uint len1 = numidx + cv4_numericbytes(cast(uint)size);
     debtyp_t *d = debtyp_alloc(len1 + cv_stringbytes(id));
     cv4_storenumeric(d.data.ptr + numidx, cast(uint)size);
-    const uint len = len1 + cv_namestring(d.data.ptr + len1,id);
```
Comment (rainers) [inline src/dmd/tocvdebug.d]: same here.

**Review by rainers — CHANGES_REQUESTED** [review]

**Comment by Geod24** [comment]
Thanks for the pointers @rainers . Fixed.

**Review by rainers — APPROVED** [review]

**Pull request merged**